packages/admin/cms-admin/src/preview/SitePreview.sc.tsx

```diff
@@ -18,32 +17,31 @@ export const StyledButton = styled(Button)`
     color: ${({ theme }) => theme.palette.common.white};
 `;
 
-export const CometLogoWrapper = styled("div")`
+export const SiteInformation = styled("div")`
     display: flex;
     align-items: center;
     color: ${({ theme }) => theme.palette.common.white};
-    text-transform: uppercase;
+    padding: 0 6px;
+    gap: 20px;
 `;
 
-export const CometSiteLinkWrapper = styled("div")`
-    margin-left: 20px;
-    display: flex;
+export const LogoWrapper = styled("div")`
+    display: inline-flex;
+    align-items: center;
+    gap: 6px;
+`;
+
+export const SiteLinkWrapper = styled("div")`
+    display: inline-flex;
+    align-items: center;
+    gap: 6px;
 `;
 
-export const CometSiteLink = styled(Link)`
-    margin-left: 6px;
+export const SiteLink = styled(Link)`
     color: ${({ theme }) => theme.palette.common.white};
-    text-transform: none;
+    text-decoration-color: ${({ theme }) => theme.palette.common.white};
```

Comment:
If this should just always be the same color as the link, you could use `currentColor`.

```suggestion
    text-decoration-color: currentColor;
```